ep/build-velox/src/get_velox.sh

```diff
@@ -17,7 +17,7 @@
 set -exu
 
 VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=update
+VELOX_BRANCH=2023_12_25
```

Comment:
I'll evaluate if using a rolling-updated branch name here could be better than the previous way in the next few days.